src/Kentico.Xperience.Zapier/Helpers/AdminUrlHelper.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Kentico.Xperience.Admin.Base;
+using Kentico.Xperience.Admin.Base.UIPages;
+using Kentico.Xperience.Admin.Headless.UIPages;
+using Kentico.Xperience.Admin.Websites.UIPages;
+
+namespace Kentico.Xperience.Zapier.Helpers;
+
+internal static class AdminUrlHelper
+{
+    internal static PageParameterValues GetWebPageParams(int webPageItemId, int websiteChannelId, string lang) => new()
+        {
+            { typeof(WebPageLayout), $"{lang}_{webPageItemId}" },
```

Comment:
We can use `new WebPageUrlIdentifier(lang, webPageItemId).ToString()` to generate parameter for `WebPageLayout`.